lib/emotions.rb

```diff
@@ -8,35 +8,35 @@
 require 'emotions/emotional'
 
 module Emotions
-  def self.configure
+  def configure
```

Comment:
I don’t think we can do that. `def self.method` in a module defines a singleton method.